estate/models/estate_property.py

```diff
@@ -0,0 +1,103 @@
+from odoo import api, fields, models
+from odoo.exceptions import UserError
+from dateutil.relativedelta import relativedelta
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char("Property Name", required=True)
+    active = fields.Boolean(default=True)
+    description = fields.Text("Description")
+    postcode = fields.Char("Postal Code")
+    date_availability = fields.Date(
+        "Availability Date",
+        copy=False,
+        default= lambda self: fields.Datetime.now() + relativedelta(months=3)
+    )
+    expected_price = fields.Float("Expected Price", required=True)
+    selling_price = fields.Float("Selling Price", readonly=True, copy=False)
+    bedrooms = fields.Integer("Number of bedrooms", default=2)
+    living_area = fields.Float("Living Area (sqm)")
+    facades = fields.Integer("Facades")
+    garage = fields.Boolean("Garage")
+    garden = fields.Boolean("Garden")
+    garden_area = fields.Float("Garden Area (sqm)")
+    buyer = fields.Many2one(
+        "res.partner",
+        string="Buyer",
+        copy=False
+    )
+    tag_ids = fields.Many2many(
+        "estate.property.tag",
+        string="Property Tag"
+    )
+    property_type_id = fields.Many2one(
+        'estate.property.type',
+        string="Property Type"
+    )
+    offer_ids = fields.One2many(
+        "estate.property.offer",
+        "property_id",
+        string="Offers"
+    )
+    salesman = fields.Many2one(
+        "res.users",
+        string="Salesman",
+        default=lambda self: self.env.user
+    )
+    garden_orientation = fields.Selection(
+        selection = [
+            ('north', 'North'),
+            ('south', 'South'),
+            ('west', 'West'),
+            ('east', 'East')
+        ]
+    )
+    state = fields.Selection(
+        selection = [
+            ('new', 'New'),
+            ('offer_received', 'Offer Received'),
+            ('offer_accepted', 'Offer Accepted'),
+            ('sold', 'Sold'),
+            ('cancelled', 'Cancelled')
+        ],
+        required=True, copy=False, default="new"
+    )
+    total_area = fields.Float("Total Area(sqm)", compute="_compute_total_area")
```

Comment:
```suggestion
    total_area = fields.Float("Total Area(sqm)", compute="_compute_total_area")
```
what are the characteristics of compute field?

Reply:
A compute field is special type of field whose value is dynamically computed using a method. Following are the key characteristics:
* Recomputed on every access
* It is not stored by Default (can be stored `store=true`)
* supports `inverse` and `search` methods